ci-operator/config/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main__acm-ocp4.15-lp-interop.yaml

```diff
@@ -49,15 +49,15 @@ tests:
-          {"step": "acm-tests-clc-create", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "ACM", "jira_component": "QE", "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp","CI","ClusterLifecycle","QE-ACM","interop-tests"], "jira_assignee": "vboulos@redhat.com", "group": {"name": "clc", "priority": 1}},
-          {"step": "acm-tests-clc-destroy", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "ACM", "jira_component": "QE", "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp","CI","ClusterLifecycle","QE-ACM","interop-tests"], "jira_assignee": "vboulos@redhat.com", "group": {"name": "clc", "priority": 3}},
-          {"step": "acm-fetch-*", "failure_type": "all", "classification": "Fetch managed cluster failure", "jira_project": "ACM", "jira_component": "QE", "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp","CI","QE-ACM","interop-tests"], "jira_assignee": "vboulos@redhat.com", "group": {"name": "clc", "priority": 2}},
-          {"step": "acm-tests-obs", "failure_type": "all", "classification": "OBS testcase failure", "jira_project": "ACM", "jira_component": "QE", "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp","CI","Observability","QE-ACM","interop-tests"], "jira_assignee": "vboulos@redhat.com"},
-          {"step": "acm-tests-grc", "failure_type": "all", "classification": "GRC testcase failure", "jira_project": "ACM", "jira_component": "QE", "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp","CI","GRC","QE-ACM","interop-tests"], "jira_assignee": "vboulos@redhat.com"},
-          {"step": "acm-tests-alc", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "ACM", "jira_component": "QE", "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp","CI","ApplicationLifecycle","QE-ACM","interop-tests"], "jira_assignee": "vboulos@redhat.com"},
+          {"step": "acm-mch", "failure_type": "all", "classification": "MCH install", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp","interop-tests"]},
+          {"step": "acm-tests-clc-create", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "ACM", "jira_component": "QE", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp","CI","ClusterLifecycle","QE-ACM","interop-tests"], "jira_assignee": "vboulos@redhat.com"},
+          {"step": "acm-tests-clc-destroy", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "ACM", "jira_component": "QE", "group": {"name": "lp-tests", "priority": 3}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp","CI","ClusterLifecycle","QE-ACM","interop-tests"], "jira_assignee": "vboulos@redhat.com"},
```

Comment:
There's a discrepancy here between this file and the file above it for acm-ocp4.14.

This step in the acm-ocp4.14 config file is a priority 2 but here it is a priority 3.

I think the priorities for this file are perfect, I would recommend to change the acm-ocp4.14 file above to match this file.